sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java

```diff
@@ -69,6 +72,12 @@ public Settings indexSettings() {
         return ImmutableSettings.builder().put("number_of_replicas", 0).build();
     }
 
+    @After
+    public void cleanUp() {
```

Comment:
would prefer to name the method to what it actually do, so it also prevent that subclasses overwrites it by accident